commands/commands_test.go

```diff
@@ -102,7 +102,7 @@ func executeWithArgs(t *testing.T, args ...string) (exitCode int, output []byte)
 	cmd.SetArgs(args)
 	cmd.Execute()
 
-	return
+	return exitCode, output
```

Comment:
This removes the naked return but not the possibility to add more.
Maybe we should also remove the names from the return values to properly fix this?

Reply:
I don't know, I can remove it in declare as a normal variable, for me that function in general is difficult to read

Reply:
Done 157bbbb6b0e39b064e5b1452648d34c42c009715